src/mingwX64Main/kotlin/com/autodesk/coroutineworker/AutoreleasePool.kt

```diff
@@ -0,0 +1,3 @@
+package com.autodesk.coroutineworker
+
+internal actual inline fun <R> autoreleasepool(block: () -> R): R = block()
```

Comment:
While this is technically available in K/N, I'm pretty sure mingw will fail to build since the required objc symbols don't exist, so on mingw, we just run the block